chips/earlgrey/src/top_earlgrey.rs

```diff
@@ -0,0 +1,3188 @@
+// Licensed under the Apache License, Version 2.0 or the MIT License.
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+// Copyright lowRISC contributors 2023.
+
+// Built for Earlgrey-M2.5.1-RC1-389-g21ce4e9761
+// https://github.com/lowRISC/opentitan/tree/21ce4e9761abdf5c919b46e5ae64a5a8e24992f7
+// Tree status: clean
+// Build date: 2023-07-20T09:36:12.725946
+
+// This file was generated automatically.
+// Please do not modify content of this file directly.
+// File generated by using template: "toplevel.rs.tpl"
+// To regenerate this file follow OpenTitan topgen documentations.
```

Comment:
I'm wondering whether it might be better to move this file into `registers`, to keep all auto-generated code contained in a single module (as it essentially only relates to registers' base offsets). However, I don't want to voice a strong opinion either way, to me the current module path is also fine.

Reply:
Moving it to registers/ seems appropriate.

Reply:
Moved.